src/main/java/codechicken/nei/recipe/RecipeCatalysts.java

```diff
@@ -85,6 +114,127 @@ public static int getRowCount(int availableHeight, int catalystsSize) {
         return NEIServerUtils.divideCeil(catalystsSize, columnCount);
     }
 
+    /**
+     * The intent for using csv is to allow catalysts to work outside GTNH.
+     * GTNH-specific or actively maintained mods can add their catalysts through API.
+     * I hope nobody will have a terrible idea of adding all GT machines by csv...
```

Comment:
Lol.  The reason it was added to handler.csv is so that it'd work on any version of GT without needing the newest version that registers them ;)